src/model/MultiValueColumn.ts

```diff
@@ -50,33 +50,36 @@ export class CustomSortCalculation {
   }
 }
 
-// function getPercentile(data, percentile) {
-//
-//       var index = (percentile / 100) * data.length;
-//       var result;
-//       if (Math.floor(index) === index) {
-//         result = (data[(index - 1)] + data[index]) / 2;
-//       } else {
-//         result = data[Math.floor(index)];
-//       }
-//       return result;
-//     }
-
 function numSort(a, b) {
   return a - b;
 }
 
+enum Sort {
+
+  min, max, median, q1, q3
+}
+
+
+interface boxPlotData {
```

Comment:
interfaces start with a captial `I` followed by camelCase by convetion. In this example `IBoxPlotData` + since the items are readonly, mark them as readonly